lib/foreman_remote_execution/engine.rb

```diff
@@ -244,6 +244,12 @@ class Engine < ::Rails::Engine
           parent: :monitor_menu,
           after: :audits
 
+        menu :labs_menu, :job_invocations_detail,
+          url_hash: { controller: :job_invocations_detail, action: :index },
```

Comment:
If an `url` is provided, then the `url_hash` is used only when checking permissions when deciding whether the menu item should be shown. Since there is no job_invocations_detail controller with index action, this won't show for a non-admin user even with full set of roles. Mapping this to permissions of an existing controller with established permissions should work
```suggestion
          url_hash: { controller: :job_invocations, action: :show },
```